### DISCUSSION TITLE
Update Stacking Group

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1431] [Proposals]
### Goal

Discutir e avaliar a possibilidade de refletir o comportamento do componente que está no código na biblioteca do Figma.

### Context

No fluxo de Seguros, temos a necessidade de espelhar o comportamento do StackingGroup que está no código no Figma. Nesse caso, estamos usando o Stacking Group no advancedCard.

A evolução no Figma seria:
- Ter a opção de carregar um icon;
- Ter a opção de carregar um circle icon;
- Ter a opção de ter apenas 1 elemento;
- Alterar tamanho com múltiplos de 8.

### Impact

Fazer essa evolução no componente, vai nos possibilitar aplicar o advancedCard no fluxo de Seguros ao invés de criar cards novos, dessa forma, podemos reduzir o tempo de desenvolvimento e também não aumentaria o número de componentes em Extended ou Mística.

### Discussion Questions

ND

### Do you have a visual proposal?

https://www.figma.com/file/o6e4TMmc3InCthqcsCO8cZ/Evolu%C3%A7%C3%A3o-Stacking-Group?type=design&node-id=0%3A1&mode=design&t=4bk3PTGROm7eQHXs-1

**Comment by yceballost** [comment]
Olá Anne!

We are going to include this improvements in Figma

For the last point, "Alterar tamanho com múltiplos de 8", we want to avoid to include a tones of variants with different sizes for stackingGroup because the maintenance of that can be very large. We are going to explore an alternative to change the size from the component.

https://github.com/Telefonica/mistica-design/issues/1444

**Reply by annefranca** [reply]
@yceballost 

Você teria uma estimativa de prazo? Dessa forma conseguimos nos organizar internamente com os entregáveis.

**Reply by aweell** [reply]
Hi @annefranca,

If only a set of limited sizes could be used in the Figma component, what sizes would you need to use in the component (example: 40, 48, 64)?

**Reply by annefranca** [reply]
@aweell podemos iniciar com 24 e finalizar com 64, ou seja, teríamos disponível os tamanhos 24, 32, 40, 48, 56 e 64?

**Reply by AnaMontes11** [reply]
@annefranca 
While we are studying how to include sizes multiples of 8 in Figma, in release 9.3.0 we have included the following improvements:

- Option with circle icon
- Option with only one element

**Reply by annefranca** [reply]
@AnaMontes11 existe algum tipo de complexidade para não implementar a alteração de tamanho?

**Reply by aweell** [reply]
Hi @annefranca, we are searching a way to avoid 6 different sizes x quantity of elements as variants of the component. Since there's no easy way we decided to release the circle icon and the only element variant and continue our efforts during 10.0.0.

**Reply by AnaMontes11** [reply]
Exactly, this request implies an exponential increase in variants within the component. We will still working on it. @annefranca

**Reply by annefranca** [reply]
@AnaMontes11 @aweell mas essa evolução de tamanhos seria apenas no Figma, pois já existe essa possibilidade em código, ou seja, seria espelhar o mesmo comportamento. https://mistica-web.vercel.app/?path=/story/components-stackinggroup--default&args=size:68

**Reply by aweell** [reply]
Yes, it would be to replicate the behaviour. Maybe we're missing something here, we would feel more than pleased if your team helps us to create a proposal that replicates web behaviour in Figma without ending with infinite (or a high number) of variants.

**Reply by annefranca** [reply]
@AnaMontes11 @aweell 

Fizemos uma proposta para simplificar a manutenção do componente incluindo as variantes de tamanho. 
Link: https://www.figma.com/file/o6e4TMmc3InCthqcsCO8cZ/Evolu%C3%A7%C3%A3o-StackingGroup?type=design&node-id=0%3A1&mode=design&t=wD4SiosvhbIJcPr1-1

Ficamos no aguardo do retorno de vocês.

@BrunoBiagioniNeto para acompanhar.

**Reply by AnaMontes11** [reply]
Thank you, @annefranca. We believe it fits very well. We will include it following your suggestion.

**Reply by annefranca** [reply]
@AnaMontes11 sabe dizer quando esse ajuste das variantes de tamanho vai estar publicado?

**Reply by AnaMontes11** [reply]
Hi @annefranca It will be in the next release. We are working on another alternative because your proposal doesn't quite fit with what we can include.
